requirements.txt

```diff
@@ -4,7 +4,9 @@ coverage==4.1
 coveralls==1.1
 decorator==4.0.10
 dj-database-url==0.4.1
+dj-static==0.0.6
```

Comment:
@andela-aoduola: I think `./manage.py collectstatic` is always run against staging and production. Do we still need `dj-static` on top that?